doom-modeline-core.el

```diff
@@ -926,6 +926,13 @@ then this function does nothing."
 (add-hook 'after-setting-font-hook #'doom-modeline-refresh-font-width-cache)
 (add-hook 'server-after-make-frame-hook #'doom-modeline-refresh-font-width-cache)
 
+(add-hook 'window-configuration-change-hook
```

Comment:
I looks like more than a workaround. The hooks will bring negative performance impacts.

Reply:
That was why I didn't send an initial PR. I'm fine with waiting until the issue is fixed in upstream emacs if you'd prefer?